data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/CsaProfileCracCreatorTest.java

```diff
@@ -290,7 +293,7 @@ public void csa92() {
 
         assertNotNull(cracCreationContext);
         Set<RemedialAction<?>> remedialActions = cracCreationContext.getCrac().getRemedialActions();
-        assertEquals(9, remedialActions.size());
+        assertEquals(9, (int) remedialActions.stream().filter(ra -> ra instanceof NetworkActionImpl).count());
         // RA17 (on instant)
         NetworkActionImpl ra17 = (NetworkActionImpl) remedialActions.stream().filter(ra -> ra.getName().equals("RA17")).findAny().get();
```

Comment:
```suggestion
        assertEquals(9, cracCreationContext.getCrac().getNetworkActions().size());
        // RA17 (on instant)
        NetworkAction ra17 = cracCreationContext.getCrac().getNetworkAction("RA17");
```
and so on...
sorry I did not see this when reviewing csa-9